packages/compiler-sfc/src/compileScript.ts

```diff
@@ -1880,6 +2000,34 @@ export function compileScript(
     scriptAst: scriptAst?.body,
     scriptSetupAst: scriptSetupAst?.body
   }
+
+  function genRuntimeEmits() {
```

Comment:
Let's move this up to below `genSetupPropsType` - it's a bit weird that this is the only function declared after the return statement (and we don't use that style anywhere in this codebase)